Bio/SeqIO/NibIO.py

```diff
@@ -43,30 +43,38 @@
 try:
     hex2bytes = bytes.fromhex  # python3
 except AttributeError:
-    hex2bytes = lambda s: s.decode('hex')  # python2
+    # python 2
+    hex2bytes = lambda s: s.decode('hex')  # noqa: E731
+
+
+def bytes2hex(b):
```

Comment:
Can you put the version check outside the definition (as before), so it happens once at import time, rather than for every function call? (Not sure how much difference this would actually make to performance)

Reply:
This is called only one or two times so I thought it would be OK. Otherwise, I have to define two methods with the same name (or noqa it).

Reply:
In this case the ``# noqa`` would make sense since the two functions implementations are for different platforms. But if the function is only called once or twice per file, it should be fine as is.